.travis.yml

```diff
@@ -364,85 +138,82 @@ jobs:
                                true
                              fi
                 - stage: "build only"
-                  env: CONF=""
-                  compiler: gcc
-                  install: true
-                  script:
-                          - ./bootstrap
-                          - ./configure --enable-helper-linux
-                          - make -j $(nproc)
-                - stage: "build only"
-                  env: CONF=""
-                  compiler: clang-3.8
-                  install: true
-                  script:
-                          - ./bootstrap
-                          - ./configure --enable-helper-linux
-                          - make -j $(nproc)
-                - stage: "build only"
-                  env: CROSS_ARCH="i386"
-                  compiler: gcc
-                  install: true
-                  script:
-                          - ./bootstrap
-                          - ./configure --enable-helper-linux $CROSS
-                          - make -j $(nproc)
-                - stage: "build only"
-                  env: CROSS_ARCH="arm64"
-                  compiler: gcc
-                  install: true
+                  env: Ubuntu14.04_arm64
                   script:
-                          - ./bootstrap
-                          - ./configure --enable-helper-linux $CROSS
-                          - make -j $(nproc)
+                          - docker run  -i -t -v `pwd`:/odp -v /lib/modules:/lib/modules -v /usr/src:/usr/src
```

Comment:
Also I don't think there is a reason now to pass `/lib/modules` and `/usr/src` inside container.

Reply:
netmap modules will require that.

Reply:
No, you will just have to  build netmap modules on host system and libs in guest. Maybe we just have to package netmap.

Reply:
Ah, there are no libs. Then it's even simpler. Just build modules on host system and provide headers into guest.